cmd/kjudge/main.go

```diff
@@ -18,7 +18,7 @@ import (
 
 var (
 	dbfile      = flag.String("file", "kjudge.db", "Path to the database file.")
-	sandboxImpl = flag.String("sandbox", "isolate", "The sandbox implementation to be used (isolate, raw). If anything other than 'raw' is given, isolate is used.")
+	sandboxImpl = flag.String("sandbox", "raw", "The sandbox implementation to be used (isolate, raw). If anything other than 'raw' is given, isolate is used.")
```

Comment:
Please revert, `raw` is an unsafe default option

Reply:
@natsukagami that was for debugging because I don't have `isolate` installed 😅